### PR TITLE
CherryPicked commits to enable upgrade to latest base image

### PR DESCRIPTION
**Pull request by aravindraja93**
PR to solve this issue : https://github.com/tiiuae/mesh_com/issues/270

### CODE REVIEW

File: modules/mesh_com/entrypoint.sh
Comment (maseabunikie) [inline modules/mesh_com/entrypoint.sh:1]: @joenpera @govindsi 
Please, take a look to this file carefully. This is taken from mesh chain changes done last year. It is possible that you don't want this on the main branch.

Comment (joenpera) [reply modules/mesh_com/entrypoint.sh:1]: This needs to be asked from @sergei-petroff.  He knows the setup which should be supported in mainline.

**Review by joenpera — APPROVED** [review]

**Review by govindsi — APPROVED** [review]

**Review by govindsi — COMMENTED** [review]
Seems workflow is still failing.

 cp: cannot stat 'modules/*.deb': No such file or directory
Error: Process completed with exit code 1.

**Review by govindsi — CHANGES_REQUESTED** [review]
Seems its still require some more changes to fix the workflow.

 cp: cannot stat 'modules/*.deb': No such file or directory
Error: Process completed with exit code 1.

**Comment by aravindraja93** [comment]
@govindsi : Debian build script updated. 

And also the deb workflow is repeating having alternate scripts for the same goal , 

These two different scripts are used for the same goal, Better to fix using the same one:

1. https://github.com/tiiuae/mesh_com/blob/main/modules/mesh_com/package.sh
2. https://github.com/tiiuae/fog-ros-baseimage/blob/main/builder/packaging/package.sh

**Review by govindsi — APPROVED** [review]
Looks good now.

**Pull request merged**